course_metadata/signals.py

```diff
@@ -0,0 +1,18 @@
+"""
+This module has definition of receivers for signals
+"""
+from django.dispatch.dispatcher import receiver
+
+from xmodule.modulestore.django import SignalHandler
+
+from .tasks import update_course_aggregate_metadata
+
+
+@receiver(SignalHandler.course_published, dispatch_uid='course_metadata')
```

Comment:
I think value of `dispatch_uid` should b `course_metadata.signals.listen_for_course_publish`